spark-atlas-connector/src/main/scala/com/hortonworks/spark/atlas/types/external.scala

```diff
@@ -31,14 +30,16 @@ import org.apache.hadoop.fs.Path
 import org.apache.hadoop.hive.ql.session.SessionState
 import org.apache.spark.sql.catalyst.catalog.{CatalogDatabase, CatalogStorageFormat, CatalogTable}
 import org.apache.spark.sql.types.StructType
-
 import com.hortonworks.spark.atlas.{AtlasClient, AtlasUtils}
-import com.hortonworks.spark.atlas.utils.SparkUtils
+import com.hortonworks.spark.atlas.utils.{JdbcUtils, SparkUtils}
 import com.hortonworks.spark.atlas.sql.streaming.KafkaTopicInformation
 
 
 object external {
   // External metadata types used to link with external entities
+  val QUALIFIED_NAME_KEY = "qualifiedName"
```

Comment:
These keys are not only used for external, but also used for all entities. 

Maybe we would want to apply changing them to constant for all entities, but then code diff will mix up with two different things. 

I feel we can do the refactor from another PR. Until then, could we leave them as is were?